doc/03-Configuration.md

```diff
@@ -0,0 +1,28 @@
+# Configuration
+
+This is how you configure your Icinga DB so it works right off the bat.
+
+## Database
+
+Select the database resource for your setup.
```

Comment:
I would assume that I have to create the DB as a resource first, right? Please explain that to me, or show a screenshot where to find this and its credentials. Also, add a table with the attributes like for Redis.